torch/_inductor/triton_heuristics.py

```diff
@@ -165,7 +165,7 @@ def __init__(
         self.device_type = (
             triton_meta["device_type"] if "device_type" in triton_meta else "cuda"
         )
-        self.gpu_device = get_interface_for_device(self.device_type)
+        self.device_interface = get_interface_for_device(self.device_type)
```

Comment:
Maybe use `self.device` as a generalization of `self.gpu_device`?

Reply:
I feel that `device_interface` is clearer as this is what it's referring to? The actual `device` is a seperate thing.